_posts/en/newsletters/2020-04-01-newsletter.md

```diff
@@ -0,0 +1,132 @@
+  [eltoo][topic eltoo] mechanism to ensure an onchain spend by the most
+  recent delegate (Carol) can take precedence over spends by previous
+  delegates (Alice and Bob), assuming the trusted third party hasn't
+  colluded with a previous delegate to cheat.
+
+    This week, Tom Trevethan [posted][trevethan statechains] to the
+    Bitcoin-Dev mailing list about about two modifications of the
+    statechain design that could allow it to be used with the current
+    Bitcoin protocol rather than waiting for proposed soft fork changes
+    such as [schnorr signatures][topic schnorr signatures] signatures
+    and [SIGHASH_ANYPREVOUT][topic sighash_noinput]:
+
+    1. Replace the eltoo mechanism (which requires either [BIP116][]
+       `SIGHASH_NOINPUT` or [bip-anyprevout][] `SIGHASH_ANYPREVOUT`)
+       with a decrementing locktime similar to that proposed for
+       [duplex micropayment channels][].  E.g., when Alice receives
+       control over a statechain UTXO, a timelock would prevent her from
+       being able to unilaterally spend it onchain for 30 days; when
+       Alice transfers the UTXO to Bob, a timelock would restrict him
+       for only 29 days---this gives a spend by Bob precedence over a
+       spend by Alice.  The downside of this approach is that delegates
+       might need to wait a long time before being able to spend their
+       funds without permission from the trusted third party.
+
+    2. Replace the 2-of-2 schnorr multisig between the trusted third
+       party and the current delegate (using an [adaptor
+       signature][scriptless scripts]) with a single-sig using [secure
+       multiparty computation][mpc].  The main downside of this approach
+       is an increased complexity that makes security review harder.
+
+    Several people replied to the thread with comments and suggested
+    alternatives.
+
+- **Mitigating differential power analysis in schnorr signatures:**
+  Lloyd Fournier started a [discussion][fournier dpa] on the Bitcoin-Dev
+  mailing list about the proposal described in [Newsletter #87][news87
+  bip340 update] to update the [BIP340][] specification for schnorr
+  signatures with a recommended nonce generation function that claims to
+  be more resistant against [differential power analysis][dpa].  Power
+  analysis means monitoring the amount of electricity a hardware wallet
+  uses when it generates different signatures in order to potentially
+  learn what private key was used (or close enough that effective brute
+  forcing becomes possible).  Fournier questioned the utility of
+  combining the private key plus the randomness using an [xor][]
+  operation rather than a more standard method of hashing the private
+  key with the randomness.
+
+    BIP340 co-author Pieter Wuille [replied][wuille dpa] with an
+    explanation: in schnorr key and signature aggregation where there's
+    a known mathematical relationship between the private keys of
```

Comment:
I think this wording is slightly wrong. There isn't a relationship between the private keys of cooperating users (each private key should be essentially random) - there's a relationship between an individual's private key and the aggregate private key.

Reply:
If there's a relationship between Alice's key and the aggregate key, and there's a relationship between Bob's key and the same aggregate key, doesn't it follow that there's a relationship between Alice's key and Bob's key?  I'll push a small edit that tries to make clear that the relationship is created by the process of aggregation rather than being fundamental.

Reply:
New wording looks good. Thanks!